app/src/main/java/net/squanchy/proximity/near/NearITComponent.java

```diff
@@ -0,0 +1,15 @@
+package net.squanchy.proximity.near;
+
+import net.squanchy.injection.ActivityLifecycle;
+import net.squanchy.injection.ApplicationComponent;
+import net.squanchy.proximity.ProximityService;
+
+import dagger.Component;
+
+@ActivityLifecycle
+@Component(modules = {NearITModule.class}, dependencies = ApplicationComponent.class)
+public interface NearITComponent {
```

Comment:
Components are activity(or fragment, or entry point)-specific, so they go by the name of their entry point, modules are specific for the functionality they're offering (e.g. proximity), this should be then named after the activity which is using it (remember, one component for activity, N modules for component). A valid component name is `HomeComponent` if you're using it in the `HomeActivity`, for example.

Same goes for the injectors.